k8s/aat/common/bsp/bulk-scan-processor.yaml

```diff
@@ -25,6 +36,45 @@ spec:
       image: hmctspublic.azurecr.io/bulk-scan/processor:prod-72f18417
       environment:
         STORAGE_BLOB_SELECTED_CONTAINER: ALL
+      testsConfig:
+        memoryLimits: "3072Mi"
+        serviceAccountName: tests-service-account
+        keyVaults:
+          bulk-scan:
+            resourceGroup: bulk-scan
+            secrets:
+              storage-account-primary-key: TEST_STORAGE_ACCOUNT_KEY
+              storage-account-name: TEST_STORAGE_ACCOUNT_NAME
+              processed-envelopes-queue-send-connection-string: PROCESSED_ENVELOPES_QUEUE_WRITE_CONN_STRING
+          s2s:
+            resourceGroup: rpe-service-auth-provider-aat
+            secrets:
+              microservicekey-bulk-scan-processor-tests: TEST_S2S_SECRET
+        environment:
```

Comment:
there 2 environments. one on the above

Reply:
One is for the service itself, this one is for tests.

Reply:
I see them in the same level?